src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceDefinition.scala

```diff
@@ -39,31 +41,45 @@ object InstanceDefinition {
       createdTime: Long,
       lastUpdatedTime: Long,
       deletedTime: Option[Long],
+      version: Option[Long],
       properties: Option[Map[String, Map[String, Map[String, InstancePropertyValue]]]],
       startNode: DirectRelationReference,
       endNode: DirectRelationReference
   ) extends InstanceDefinition {
     override val instanceType: InstanceType = InstanceType.Edge
   }
 
-  implicit val nodeDefinitionEncoder: Encoder[NodeDefinition] = Encoder.forProduct6(
+  implicit val nodeDefinitionEncoder: Encoder[NodeDefinition] = Encoder.forProduct8(
     "instanceType",
     "space",
     "externalId",
     "createdTime",
     "lastUpdatedTime",
+    "deletedTime",
```

Comment:
Not sure why it was missing in encoders before, but otoh seems that *Definition is only read from responses and not used to form requests so maybe encoders aren't used.
In either case, shouldn't lead to anything big changing, worst case can be tweaked later on + adding code comments on why it is excluded if it will be (might also be the case we just need 2 separate classes)